tests/services/analysis_starter/test_balsamic_models.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import pytest
+
+from cg.constants import SexOptions
+from cg.services.analysis_starter.configurator.models.balsamic import BalsamicConfigInput
+
+
+@pytest.fixture
+def balsamic_config_input(tmp_path: Path) -> BalsamicConfigInput:
```

Comment:
I am still a fan of putting fixtures in the fixture directory 😄

Reply:
Not sure where to put it. I feel like it won't be used in any other test. Discuss